IDEAS/Buildings/Data/Materials/Screed.mo

```diff
@@ -1,11 +1,17 @@
 within IDEAS.Buildings.Data.Materials;
 record Screed = IDEAS.Buildings.Data.Interfaces.Material (
-    k=0.6,
+    k=0.45,
     c=840,
     rho=1100,
     epsLw=0.88,
-    epsSw=0.55) "Light cast concrete" annotation (Documentation(info="<html>
+    epsSw=0.55) "Typical (Belgian) screed" annotation (Documentation(info="<html>
 <p>
-Thermal properties of screed.
+Thermal properties of screed. The values are based on those provided in <a href=\"https://www.rsprojects.eu/images/Duremit/Warmtegeleiding%20van%20Duremitchape.pdf\">this document</a>.
```

Comment:
Heb je nog bronnen vergeleken?

Reply:
@Mathadon I did a new search on the web, where I focused on 'cement screed' (other types exist like liquid screed, anhydrite screed, insulating screed, ... each with its own specific thermal properties). I found the following data:
- rho=1900, k=1.2
https://www.rsprojects.eu/images/Duremit/Warmtegeleiding%20van%20Duremitchape.pdf
- k=1.3
https://www.joostdevree.nl/shtmls/warmtegeleidingscoefficient.shtml
- rho=2000, k=1.4
https://buildex.techinfus.com/en/uteplenie/teploprovodnost-uteplitelej.html
- rho=2100, k=1.95, c=900
Rauch, Michael & Werther, Norman & Winter, Stefan. (2020). FIRE DESIGN METHODS FOR TIMBER FLOOR ELEMENTS THE CONTRIBUTION OF SCREED FLOOR TOPPINGS TO THE FIRE RESISTANCE. 10.13140/RG.2.2.16032.00004. 
- rho=1800, k=1.35, c=1000
Gagliano, Antonio & Nocera, Francesco & Detommaso, Maurizio & Agrifoglio, Antonio & Patania, Francesco. (2016). Dynamic simulations and experimental measurements on an extensive green roof in Mediterranean climate. 
- rho=1200, k=1, c=1250
Values used for the buildings of De Schipjes

Based on these resources, I would opt for the following properties:
**rho=2000, k=1.3, c=1000**

Reply:
Ok! Can you summarise the search in the documentation? Or add a permalink to your post in the documentation:
<img width="822" alt="image" src="https://user-images.githubusercontent.com/5970879/181703162-3bbf5f3b-cdec-4587-afd0-bca808a72df3.png">

Reply:
Done! Remark that the new values are close to those of concrete (rho=2240, k=1.4, c=900).

Reply:
Strange.. but ok!